rpc_jobs/rpc_aio.yml

```diff
@@ -339,6 +338,8 @@
                   kibana.kibana(env.KIBANA_SELENIUM_BRANCH)
                   holland.holland()
                 }} else if (env.STAGES.contains("Leapfrog Upgrade")) {{
+                  // Required for leapfrog resource generation
+                  tempest.tempest_install()
                   deploy.upgrade_leapfrog(environment_vars: environment_vars)
```

Comment:
What about moving it into `deploy.upgrade_leapfrog`, that should ensure it can't get removed by mistake and simplifies the template.